Makefile.PL

```diff
@@ -17,9 +17,9 @@ my %WriteMakefileArgs = (
   "MIN_PERL_VERSION" => "5.014",
   "NAME" => "Ethereum::RPC::Client",
   "PREREQ_PM" => {
-    "Future" => "0.38",
+    "Future" => "0.37",
```

Comment:
Downgrades?

Reply:
Yes, this downgrade is just to keep the same version that we have in our cpan